pkg/syncer/util.go

```diff
@@ -364,3 +384,66 @@ func SyncerInitConfigInfo(ctx context.Context) (*cnsconfig.ConfigurationInfo, er
 	}
 	return configInfo, nil
 }
+
+// initializeVolumeInfoService initialises volumeInfoService's instance
+func initializeVolumeInfoService(ctx context.Context) error {
+	// Return if volumeInfoService is already initialised
+	if volumeInfoService != nil {
+		return nil
+	}
+
+	// Initialise volumeInfoService
+	volumeInfoSvc, err := cnsvolumeinfo.InitVolumeInfoService(ctx)
+	if err != nil {
+		return err
+	}
+	volumeInfoService = volumeInfoSvc
+	return nil
+}
+
+// getVcHostAndVolumeManager returns VC host and the corresponding volume manager that can access the VC.
```

Comment:
shall we mention about volume like "getVcHostAndVolumeManager returns VC host and the corresponding volume manager that can access given volume" ?

Reply:
Done